docs/docs/configuration/authentication.md

```diff
@@ -26,6 +26,8 @@ In the event that you are locked out of your instance, you can tell Frigate to r
 
 ## Login failure rate limiting
 
+# TODO: Rui Update to use FastAPI
```

Comment:
what is changing here? because of the rate limit?

Reply:
Yes, I left the comment to remember to change the docs to say 'rate limit by XX FastAPI ...'